modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuildController.java

```diff
@@ -554,10 +563,23 @@ private static IndexStorage indexStorage(
             int partitionId,
             CatalogIndexDescriptor indexDescriptor
     ) {
-        IndexStorage indexStorage = mvTableStorage.getIndex(partitionId, indexDescriptor.id());
+        IndexStorage indexStorage;
```

Comment:
Maybe in a separate method to make it look more elegant?

Reply:
I would prefer to leave it as is, reasoning is identical to https://github.com/apache/ignite-3/pull/5725#discussion_r2068161590